statsmodels/sandbox/distributions/tests/test_multivariate.py

```diff
@@ -25,6 +30,7 @@ def setup_class(cls):
         corr2[2,1] = -0.5
         cls.corr2 = corr2
 
+    @pytest.mark.skipif(not SP_LT_116, reason="SciPy < 1.16.0 required")
```

Comment:
The reason doesn't match the condition. `not <` is `>=`

Reply:
This one is the correct one.  Skipif Not < 1.16 means that we only run for scipy < 1.16.